test/test_oj_options.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'helper'
+require 'fluent/test'
+require 'fluent/oj_options'
+
+class OjOptionsTest < ::Test::Unit::TestCase
+  setup do
+    @oj = Fluent::OjOptions.new
+  end
+
+  sub_test_case "OjOptions" do
+    test "when no env vars set, returns default options" do
+      ENV.clear
```

Comment:
Ah, now I've got the reason why so many tests are broken.
Because `ENV` is cleared here, later tests which depend on `ENV` are broken.
I'll fix it.

Reply:
Ah I see, nice catch. Thanks